spec/game_logic_spec.rb

```diff
@@ -0,0 +1,58 @@
+require './lib/game_logic'
+
+RSpec.describe 'testing game logic' do
+  context 'Player' do
+    it 'returns player1 mark' do
+      player1 = Player.new('player1')
+      expect(player1.mark).to eql('+')
+    end
+
+    it 'returns player2 mark' do
+      player2 = Player.new('player2')
+      expect(player2.mark).to eql('-')
+    end
+  end
+
+  context 'Board' do
+    let(:board) { Board.new }
+    describe '#space_selection' do
+      it 'returns a new board with a changed position' do
+        expect(board.space_selection(4, '+', [])).to eq([4])
+      end
+    end
```

Comment:
Context should be inside a describe block, not the other way around.  See guidelines [here](https://github.com/rubocop-hq/rspec-style-guide#context-cases).